service/history/replicatorQueueProcessor.go

```diff
@@ -461,21 +462,22 @@ func (p *replicatorQueueProcessorImpl) getTasks(
 		}
 	}
 
-	// Note this is a very rough indicator of how much the remote DC is behind on this shard.
-	p.metricsClient.RecordTimer(
+	replicationScope := p.metricsClient.Scope(
 		metrics.ReplicatorQueueProcessorScope,
+		metrics.InstanceTag(strconv.Itoa(p.shard.GetShardID())),
```

Comment:
We may need to check gc and cpu usage for this shardID tag. Not sure if it will cause the same thing as adding domain tag to task processing.

Reply:
I don't think replication is this time-sensitive. But we can verify